src/main.rs

```diff
@@ -168,6 +174,9 @@ enum CliCommand {
         #[structopt(name = "name")]
         /// name of the command in pndev.toml
         name: Option<String>,
+
+        /// optional params for the run command
+        params: Vec<String>,
```

Comment:
`arguments` in command, `params` here. choose one? or are these lexically different?

Reply:
yeah let's rename